analyses/fusion-summary/01-fusion-summary.Rmd

```diff
@@ -44,6 +44,30 @@ filterFusion <- function(df, bioid, fuses, genes) {
   return(df %>% select(Sample, FusionName))
 }
 
+#create function specifically for lgat, since we want more information from the putative oncogenic file
+filterLGATFusion <- function(df, bioid, fuses, genes) {
```

Comment:
The bioid is pulling `Sample` which harbor a fusion or gene from the list from the putative_oncogenic file (lines 24-26)
```
filterFusion <- function(df, bioid, fuses, genes) {
  if (!missing(bioid)) {
    df <- filter(df, Sample %in% bioid)
  }
```

>It looks like the main difference is the select() step? Is this right? Since this function is only used once maybe we can just use filterFusion and add a select step after that (no function).

This is correct, but `select()` cannot be used after the function because the function itself has its own select step in line 44 
```
return(df %>% select(Sample, FusionName))
}	
```

What I could do is add the extra columns into the original function so that there is only one function, and subset out the sample and FusionName with those other histologies in the end. That would get rid of the other function if that sounds good to you.

Reply:
consolidated functions in [this commit](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/830/commits/6a74fd707742fc0df05d42836ab41be570c66156)